.gitmodules

```diff
@@ -33,3 +33,6 @@
 [submodule "rts/lib/lunasvg"]
 	path = rts/lib/lunasvg
 	url = https://github.com/sammycage/lunasvg.git
+[submodule "recoil-lua-library"]
```

Comment:
This submodule shouldn't be necessary

Reply:
What about all the Lua in this project? e.g. `cont/**/*.lua`

I understand that's not actively used by BAR, but while it's in the project does it not make sense for it to have editor support?

Reply:
Let's get to that stage later, I have some ideas on how to deal with this. But this is already too complex right now. Also when we do that we'll want to have it in a folder inside `cont/springcontent` probably with its own `.luarc` and a the submodule pointing to a `.lua_library` folder or similar.

Reply:
Spoke more to this here: https://github.com/beyond-all-reason/spring/pull/1775#discussion_r1933042780

Reply:
Submodule has been removed and all workflows updated.